.tool-versions

```diff
@@ -0,0 +1 @@
+python 3.9.17
```

Comment:
Added this since asdf is being used on https://github.com/hackgvl/slack-events-bot